src/RealmJS.xcodeproj/project.pbxproj

```diff
@@ -1226,9 +1235,16 @@
 			buildSettings = {
 				DEBUG_INFORMATION_FORMAT = dwarf;
 				HEADER_SEARCH_PATHS = (
-					"$(SRCROOT)/object-store/src",
-					"$(SRCROOT)/../vendor",
-					"$(SRCROOT)/../vendor/core/include",
+					"$(SRCROOT)/object-store/src/**",
+					"$(SRCROOT)/../vendor/**",
+					"$(SRCROOT)/../vendor/core/include/**",
+				);
+				"HEADER_SEARCH_PATHS[arch=*]" = (
```

Comment:
I wonder if this is the same as `HEADER_SEARCH_PATHS`?

Reply:
I don't know